src/tests/ftest/control/ms_resilience.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from __future__ import print_function
+
+from apricot import TestWithServers
+from general_utils import stop_processes
+import random
+import socket
+import time
+
+
+class ManagementServiceResilience(TestWithServers):
+    """Test Class Description:
+    Verify that MS is accessible as long as one instance is left.
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Inititialize a ManagementServiceResilience object."""
+        super(ManagementServiceResilience, self).__init__(*args, **kwargs)
+        self.setup_start_servers = False
+        self.dmg = self.get_dmg_command()
+
+    def test_ms_resilience(self):
+        """
+        JIRA ID: DAOS-3798
+
+        Test Description:
+            Test management service is accessible after instances are removed.
+
+        :avocado: tags=all,full_regression,large,control,ms_resilience
+        """
+        # Start up 2 servers as access_points, so MS will be replicated
```

Comment:
Comment says "2 servers", but as we've discussed it should be 3 (and it does look like you've adjusted that on the next line).